flocx_market/api/app.py

```diff
@@ -1,7 +1,10 @@
 from flask import Flask
 from flask_restful import Api
 from flocx_market.api.offer import Offer
+from flocx_market.api.bid import Bid
 from flocx_market.api.root import Root
+from flask import Blueprint
+from flask import jsonify
```

Comment:
I don't think we need these two imports

Reply:
@tzumainn Those now show up in the CI tests (since we merged 7ccb4892754532014286da63c5e83b5a6bd0e064).

Reply:
And now the tests are failing :D

Reply:
Someone create a test_offer_api that is failing @tzumainn @larsks

Reply:
Odd, it's not failing for me (or for the CI) - is it possible there's some sort of merge conflict?

Reply:
@tzumainn I fixed it, now it passes

Reply:
also I think if it is communicating with the db, it should be an integration test and not a unit test.

Reply:
Agreed. None of the app tests are communicating with the db, are they?

Reply:
@tzumainn in tests/unit/db/sqlalchemy/test_offer_api.py all of them are

Reply:
Ah, but that's not an app test (testing the REST API). That's testing the database API.